src/app/nav-bar/nav-bar.component.css

```diff
@@ -0,0 +1,37 @@
+
+* {
+  margin: 0;
+  padding: 0;
+
+  font-family: Georgia;
+}
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  width: 90%;
+
+  margin: 0 auto;
+}
+
+#Navigation {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100px;
+  background: url(C:\Users\fcelta\WebstormProjects\AlorFE\src\assets\img\what-the-hex.png);
```

Comment:
The code crashes on other devices: 
```
./src/app/nav-bar/nav-bar.component.css
Module Error (from ./node_modules/postcss-loader/src/index.js):
(Emitted value instead of an instance of Error) CssSyntaxError: C:\Users\aneuhaeuser\Documents\Onbordingphase\Alor-Frontend\src\app\nav-bar\nav-bar.component.css:24:19: Can't resolve '/Users/fcelta/WebstormProjects/AlorFE/src/assets/img/what-the-hex.png' in 'C:\Users\aneuhaeuser\Documents\Onbordingphase\Alor-Frontend\src\app\nav-bar'

  22 |   width: 100%;
  23 |   height: 100px;
> 24 |   background: url(C:\Users\fcelta\WebstormProjects\AlorFE\src\assets\img\what-the-hex.png);
     |                   ^
  25 | }
  26 | 
```